cdk/kittyhawk/README.md

```diff
@@ -3,54 +3,114 @@
 Kittyhawk is the automated Kubernetes YAML generator for Penn Labs. 
 With Kittyhawk, you can define an application's deployment configuration in Typescript using objects called [constructs](https://cdk8s.io/docs/v1.0.0-beta.3/concepts/constructs/).
 
-## Getting Started
+# Getting Started
 
-The easiest way to get started is by creating a `k8s` folder in your project, and copying the [Kittyhawk starter code](https://github.com/pennlabs/kittyhawk-template) inside. You will also need to add the circleCI orb (WIP) to your ```.circleci/config.yml```
+The easiest way to get started with a Kittyhawk project is by following the following steps or by copying one of our existing products. At the end of this README, you can also see a simple example typescript file using Kittyhawk.
 
-To initialize the project, run ```yarn install``` 
+## 1. Set up CDK8s Typescript Project
+Create a `k8s` folder within your project. Within that folder, create a `cdk` folder. 
```

Comment:
We don't need a `cdk` folder within the `k8s` one, right?

Reply:
Oh, I guess we can just straight up initialize cdk8s inside `k8s` yeah